### DISCUSSION TITLE
ppstructure版面识别，可以将识别出来的各个框内的文本保存下来吗？

### DISCUSSION THREAD

**Discussion by gggdroa** [PaddlePaddle/PaddleOCR#12677] [Q&A] [answered]
这张图是版面分析后的图，在分析的时候是否可以直接将每个框里的文本保存下来呢？另外右侧出来英文字母有识别错的，这块是否包含文本识别呢？
![result1234t](https://github.com/PaddlePaddle/PaddleOCR/assets/106646583/972f57dc-5f5e-4696-9f3f-4c911ef1f7f3)

**Comment by tink2123** [comment, marked as answer]
PP-Structure 会对非 table 的区域进行OCR识别。可以看到上面的可视化也是如此。

表格的部分会单独保存成excel，如果希望端到端的输出预测结果，可以使用recover的功能，转到Word上查看结果，添加参数：
`--recovery=true --use_pdf2docx_api=true`

**Comment by gggdroa** [comment]
> PP-Structure 会对非 table 的区域进行OCR识别。可以看到上面的可视化也是如此。
> 
> 表格的部分会单独保存成excel，如果希望端到端的输出预测结果，可以使用recover的功能，转到Word上查看结果，添加参数： `--recovery=true --use_pdf2docx_api=true`

感谢回复      尝试过 会缺内容，出现少行和排本的问题  是有pdf格式(单列or多列)的限制吗？还是当前的精度确实会出现这样的情况呀？

**Comment by tink2123** [comment]
当前版面的效果有限，确实在恢复的时候还不能做到完全正确

**Comment by gggdroa** [comment]
> 当前版面的效果有限，确实在恢复的时候还不能做到完全正确

好的明白了 感谢

**Comment by gggdroa** [comment]
> 当前版面的效果有限，确实在恢复的时候还不能做到完全正确

可以直接使用pdf文件进行  版面识别吗？
想提取出来眉头和眉脚

**Comment by CapitaineNemo** [comment]
> > 当前版面的效果有限，确实在恢复的时候还不能做到完全正确
> 
> 可以直接使用pdf文件进行 版面识别吗？ 想提取出来眉头和眉脚

请问pp Structure可以做到将论文中的的文字表格切出来吗？在利用其他的工具单独表格，图像保存为图片？

**Comment by CapitaineNemo** [comment]
> > > 当前版面的效果有限，确实在恢复的时候还不能做到完全正确
> > 
> > 
> > 可以直接使用pdf文件进行 版面识别吗？ 想提取出来眉头和眉脚
> 
> 请问pp Structure可以做到将论文中的的文字表格切出来吗？在利用其他的工具单独表格，图像保存为图片？

明白了，我直接用的pp ocr，其实pp Structure有这个区分功能。
